pkg/metrics/metrics_test.go

```diff
@@ -0,0 +1,82 @@
+package metrics
```

Comment:
Top level comment: maybe worth adding very basic tests for the `== nil` cases?

Reply:
✅